### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_CreateNewProject_ScreenReader: NVDA announces incorrect label for "Name" edit field

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4775]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev2;-SH_AzureDataStudioSQL_Feb2021;-SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;-WCAG3.3.2;-JAWS;-NVDA;-STP;

--- Environment Details:
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10
Screen reader: NVDA, Jaws

--- Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Start JAWS and verify issue.
5. Tab till "Name" edit field and check what nvda announces

--- Actual:
NVDA announces incorrect label "Name. Please fill out this field" for the edit box.

Same issue is also observed with Jaws

--- Expected:
NVDA should announce correct label for edit field which is "Name" only.

--- User Impact:
Screen reader user will not get correct information about the edit field
